ui/raidboss/data/06-ew/raid/p1s.ts

```diff
@@ -176,30 +231,70 @@ const triggerSet: TriggerSet<Data> = {
     {
       id: 'P1S Powerful Light/Fire',
       type: 'GainsEffect',
       netRegex: NetRegexes.gainsEffect({ effectId: '893', capture: true }),
-      alertText: (_data, matches, _output) => {
-        if (matches.count === '14C')
-          return _output.light!();
-        return _output.fire!();
+      preRun: (data, matches) => {
+        if (matches.count === '14C') {
+          data.safeColor = 'light';
+          return;
+        }
+        data.safeColor = 'fire';
       },
+      alertText: (data, _matches, output) => data.safeColor && output[data.safeColor]!(),
+      outputStrings: fireLightOutputStrings,
+    },
+    {
+      id: 'P1S Shackles of Time on You',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'AB5' }),
+      condition: Conditions.targetIsYou(),
+      infoText: (_data, _matches, output) => output.text!(),
+      // For raid calling purposes, this might be useful but don't clutter TTS.
+      tts: null,
       outputStrings: {
-        fire: {
-          en: 'Stand on fire',
-          de: 'Auf der Feuerfläche stehen',
-          fr: 'Placez-vous sur le feu',
-          cn: '站在火',
-          ko: '빨간색 바닥 위에 서기',
+        text: {
+          en: 'Shackles of Time on YOU',
         },
-        light: {
-          en: 'Stand on light',
-          de: 'Auf der Lichtfläche stehen',
-          fr: 'Placez-vous sur la lumière',
-          cn: '站在光',
-          ko: '흰색 바닥 위에 서기',
+      },
+    },
+    {
+      id: 'P1S Shackles of Time',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'AB5' }),
+      alertText: (data, matches, output) => {
+        if (!data.safeColor)
+          return;
+        if (matches.target === data.me)
+          return output[data.safeColor]!();
+        return output[data.safeColor === 'fire' ? 'light' : 'fire']!();
+      },
+      outputStrings: fireLightOutputStrings,
+    },
+    {
+      id: 'P1S Fourfold Shackles of Companionship',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: ['B4[567]', 'B6B'] }),
+      condition: Conditions.targetIsYou(),
+      durationSeconds: (_data, matches) => parseFloat(matches.duration),
+      alertText: (_data, matches, output) => output.colorAndNumber!({ num: shackleMapping[Math.ceil(parseFloat(matches.duration))] }),
+      outputStrings: {
+        colorAndNumber: {
+          en: 'Close #${num}',
+        },
+      },
+    },
+    {
+      id: 'P1S Fourfold Shackles of Loneliness',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: ['B4[89A]', 'B6C'] }),
+      condition: Conditions.targetIsYou(),
+      durationSeconds: (_data, matches) => parseFloat(matches.duration),
+      alertText: (_data, matches, output) => output.colorAndNumber!({ num: shackleMapping[Math.ceil(parseFloat(matches.duration))] }),
+      outputStrings: {
+        colorAndNumber: {
+          en: 'Far #${num}',
```

Comment:
Can you break these out into four output strings for each duration, so folks can override the text with the marker they have to go to potentially?